sea-orm-codegen/src/entity/column.rs

```diff
@@ -143,6 +143,30 @@ impl Column {
         }
         col_def
     }
+
+    pub fn get_info(&self) -> String {
+        let type_info = self.get_rs_type().to_string().replace(" ", "");
+        let key_info = self.key_info();
+        if key_info.is_empty() {
+            return format!("Column `{}`: {}", self.name, type_info);
+        } else {
+            return format!("Column `{}`: {}, {}", self.name, type_info, key_info);
+        }
+    }
+
+    fn key_info(&self) -> String {
+        let mut vec: Vec<&str> = vec![];
+        if self.auto_increment {
+            vec.push("auto_increment")
+        }
+        if self.not_null {
+            vec.push("not_null")
+        }
+        if self.unique {
+            vec.push("unique")
+        }
+        return vec.join(", ");
+    }
```

Comment:
Just a nitpick, I think we and refactor the code and use `write!` macro instead of `join` on an `Vec`?
- https://doc.rust-lang.org/std/macro.write.html#examples

Reply:
I refactored in [this commit](https://github.com/SeaQL/sea-orm/pull/735/commits/b57e17269a1a5416cc13ba04691ac1911a91a555)